nixos/modules/services/hardware/pid-fan-controller.nix

```
@@ -0,0 +1,163 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.pid-fan-controller;
+  heatSource = {
+    options = {
+      name = lib.mkOption {
+        type = lib.types.uniq lib.types.nonEmptyStr;
+        description = "name of heat source";
+      };
+      wildcard_path = lib.mkOption {
```

```suggestion
      wildcardPath = lib.mkOption {
```
camelCase everything, please.